### PR TITLE
test: Mock tests for SshnpCore

### PR DESCRIPTION
**Pull request by xavierchanth**
**- What I did**

- Stubbed and mocked the constructor + initialization of SshnpCore

**- How I did it**

**- How to verify it**

**- Description for the changelog**
test: Mock tests for SshnpCore

### CODE REVIEW

File: packages/noports_core/test/sshnp/sshnp_core_constants.dart [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class TestingKeyPair {
```
Comment (gkc) [inline packages/noports_core/test/sshnp/sshnp_core_constants.dart:1]: Why not just generate at test time?

**Review by gkc — DISMISSED** [review]
Generally I prefer to generate keypairs etc in test code rather than hard-coding but otherwise LGTM, nice work

**Comment by xavierchanth** [comment]
Can I get another review @gkc? Had to address a merge conflict.

File: packages/noports_core/test/sshnp/sshnp_core_constants.dart (hunk 1 above)
Comment (xavierchanth) [reply packages/noports_core/test/sshnp/sshnp_core_constants.dart:1]: I can open another PR which generates the temp keys later on, it was quicker for me to hard code.

**Review by gkc — APPROVED** [review]

**Pull request merged**